p2p/pex/pex_reactor.go

```diff
@@ -264,7 +286,11 @@ func (r *PEXReactor) ReceiveAddrs(addrs []*p2p.NetAddress, src Peer) error {
 
 	srcAddr := src.NodeInfo().NetAddress()
 	for _, netAddr := range addrs {
+		// TODO: make sure correct nodes never send nil and return error
+		// TODO(melekes): punish for incorrect data
+		//   if a netAddr == nil
 		if netAddr != nil && !isAddrPrivate(netAddr, r.config.PrivatePeerIDs) {
+			// TODO(xla): Move the list to Addressbook and call it something like `blacklist_ids`.
```

Comment:
@xla whats this about ?

Reply:
oh move PrivatePeerIDs to AddrBook ? I agree we should probably do that, but they're not really "blacklisted". They're just kept private

Reply:
Was part of a discussion about boundaries between pex and addressbook, the name was just ad-hoc and as example of how it could look like.